build.sh

```diff
@@ -19,62 +19,14 @@ if [[ "$SHOULD_BUILD" == "yes" ]]; then
   echo "LATEST_MS_COMMIT: ${LATEST_MS_COMMIT}"
   echo "BUILD_SOURCEVERSION: ${BUILD_SOURCEVERSION}"
 
-  cp -rp src/* vscode/
-  cd vscode || exit
-
   export npm_config_arch="$BUILDARCH"
   export npm_config_target_arch="$BUILDARCH"
```

Comment:
Maybe these exports only apply to the `yarn` step and could be removed, but I left them here also just in case.